integrations/github/src/haystack_integrations/components/connectors/github/repo_forker.py

```diff
@@ -41,7 +41,7 @@ class GitHubRepoForker:
     def __init__(
         self,
         *,
-        github_token: Secret = Secret.from_env_var("GITHUB_TOKEN"),
+        github_token: Optional[Secret] = None,
```

Comment:
I believe the change here needs to be reverted
```suggestion
        github_token: Secret.from_env_var("GITHUB_TOKEN"),
```